pages/test-evaluate/evaluation-tools/selecting-eval-tools.pl.md

```diff
@@ -0,0 +1,230 @@
+    <td>Lista dokumentów z rozwijanym polem wyboru i ikoną filtra, pozostają tylko dwa dokumenty.</td>
+  </tr>
+  <tr>
+    <td>Z narzędziami i wiedzą, jak z nich prawidłowo korzystać, masz wszystko, czego potrzebujesz, aby skuteczniej znajdować problemy z dostępnością.</td>
+    <td>Osoba,przy niej ikony narzędzi i żarówki. Ikony łączą się ze sobą, tworząc szkło powiększające ze słowem „bariery”.</td>
+  </tr>
+  <tr>
+    <td>Dostępność internetowa: niezbędna dla niektórych, przydatna dla wszystkich.</td>
+    <td>Wokół komputera układają się ikony: ręki, oka, mózgu, ucha i ust z falą dźwiękową.</td>
+  </tr>
+  <tr>
+    <td>Aby uzyskać więcej informacji o narzędziach oceny dostępności, odwiedź stronę w3.o-r-g/W-A-I/evaluation.</td>
+    <td>Narzędzia oceny, logo W3C i Inicjatywy na Rzecz Dostępności Internetu (WAI).</td>
+  </tr>
+</tbody>
+</table>
+
+{% include excol.html type="end" %}
+
+## Co mogą zrobić narzędzia oceny, a *czego nie potrafią*
+{:#cannot}
+
+Narzędzia oceny dostępności stron internetowych mogą pomóc w szybkim rozpoznaniu potencjalnych barier dostępności. Można z nich korzystać na wszystkich etapach procesu projektowania i tworzenia stron internetowych. Narzędzia mogą zapewniać w pełni zautomatyzowane testy i pomagać w ręcznej ocenie.
+
+Automatyczne narzędzia nie mogą sprawdzić wszystkich aspektów dostępności. Wymagany jest ludzki osąd. Czasami narzędzia oceny mogą dawać fałszywe lub wprowadzające w błąd wyniki. Narzędzia oceny dostępności stron internetowych nie mogą jednoznacznie ocenić dostępności, mogą jedynie w tym pomóc.
+
+
+
+## Własności narzędzi oceny
+{:#features}
+
+Narzędzia oceny dostępności cyfrowej są skierowane do różnych odbiorców. Korzystają z nich projektanci, programiści, nietechniczni autorzy treści, testerzy jakości, a czasem także użytkownicy. Narzędzia oferują różne możliwości i funkcje, które użytkowniy mogą porównać i ocenić, aby dobrać narzędzia pod kątem swoich konkretnych potrzeb.
+
+
+
+### Główne zastosowania
+
+Poniższe własności opisują narzędzia znajdujące się na [[Liście narzędzi oceny dostępności cyfrowej]](/test-evaluate/tools/list/). Można ich używać jako filtrów, aby zmniejszyć liczbę wyświetlanych narzędzi.
+
+**Cel**
+: Czy szukasz narzędzia, które wykona zautomatyzowany test, czy też narzędzie powinno pomóc Ci w ręcznym testowaniu lub symulacji doświadczenia użytkownika.
+
+**Oceniany produkt**
+: Zależnie od rodzaju produktu, który oceniasz (np. strona internetowa, dokument, aplikacja), możesz potrzebować konkretnego narzędzia, które będzie Ci najbardziej pomocne. Niektóre narzędzia są przystosowane do oceny stron internetowych, inne do dokumentów, aplikacji komputerowych, aplikacji mobilnych itp.
+
+**Płatne lub bezpłatne**
+: Narzędzia oceny są dostępne w ramach różnych typów licencji, takich jak licencja Open Source, komercyjna, korporacyjna itp. Niektóre narzędzia są płatne, inne darmowe.
+
+**Zasada działania**
```

Comment:
@zwiastunsw @viking-pl This term seems more specific than "Type of tool", isn't it?